.github/workflows/test.yml

```diff
@@ -174,3 +175,77 @@ jobs:
           cp functionalAccounts.json.test functionalAccounts.json
           docker-compose up --build -d
           npm run test:api
+
+  generate_and_test_openapi_client:
+    name: Generate and test OpenAPI Client libraries
+    needs: [install-and-cache]
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checkout your code
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+
+      - name: Restore node_modules
+        id: cached-node-modules
+        uses: actions/cache@v3
+        with:
+          path: node_modules
+          key: node-modules-${{ hashFiles('package-lock.json') }}-${{ env.NODE_VERSION }}
+
+      # Generate OpenAPI definition
+      - name: Start Backend for later tests and generate openapi.json
```

Comment:
should run the  version without elastic-search

Reply:
I did disable it with the environment variable. What are you referencing / how do I use the version without elastic-search?

Reply:
That just tells scicat not to use elastic search , but still fires up es from the docker-compose file. 
From the docker-compose yuo should only need mongodb ( so a docker-compose up -d mongodb should be sufficient